### DISCUSSION TITLE
Getting error on elasticsearch

### DISCUSSION THREAD

**Discussion by gbigger** [Security-Onion-Solutions/securityonion#7751] [Unsupported Versions]
Upgraded to the latest version 2.3.91, no neither my search node nor my forwarding node are connected to grid.  I am see that elasticsearch is not starting even if i manually start it.  This is the error I am receiving and I have no idea what it means.

local:
    Data failed to compile:
----------
    Rendering SLS 'base:elasticsearch' failed: Jinja error: 'NoneType' object is not iterable
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/salt/utils/templates.py", line 502, in render_jinja_tmpl
    output = template.render(**decoded_context)
  File "/usr/lib/python3.6/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/usr/lib/python3.6/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/usr/lib/python3.6/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "<template>", line 146, in top-level template code
TypeError: 'NoneType' object is not iterable

; line 146

---
[...]
    - user: 930
    - group: 939
    - template: jinja

#sync templates to /opt/so/conf/elasticsearch/templates
{% for TEMPLATE in TEMPLATES %}    <======================
es_template_{{TEMPLATE.split('.')[0] | replace("/","_") }}:
  file.managed:
    - source: salt://elasticsearch/templates/{{TEMPLATE}}
    {% if 'jinja' in TEMPLATE.split('.')[-1] %}
    - name: /opt/so/conf/elasticsearch/templates/{{TEMPLATE.split('/')[1] | replace(".jinja", "")}}
[...]
---

**Comment by dougburks** [comment]
Here's another user that had an ``object is not iterable`` error, take a look and see if it might help you:
https://github.com/Security-Onion-Solutions/securityonion/discussions/7646
